Task10.py

```diff
@@ -0,0 +1,52 @@
+"""
+    Task 10.
+
+    Description:
+
+Given the list of integers ( positive , negative, zeros)
+Find max multiplication of three values in list
+
+Input: [-1, 1, 2, 0, 3, 12, 4, 5, -1, 6]
+Output: Max value = "360". Nums are: (12, 5, 6)
+
+Input: [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+Output: Max value = "2016". Nums are: (-7, 12, -24)
+"""
+
+l1 = [-1, 1, 2, 0, 3, 12, 4, 5, -1, 6]
+
+
+# creating a copy of the list for further modification
+l2 = l1.copy()
+
+# sorting modified list
+list_sorted = sorted(l2)
+
+# getting 3 values from each boundary
+max_positive_list = list_sorted[-3:]
+max_negative_list = list_sorted[:3]
+
+# checking 1-st option of getting max value
+multiply_only_positive = (max_positive_list[0] *
```

Comment:
there could be only two options  for maximum
`el[0] * el[1] * el[-1]` from sorted list or 
`el[-3] * el[-2] * el[-1]` from sorted list
other operations seems a little bit complicated